src/settings/LostItemFeePolicy/components/ViewSections/LostItemFeeSection/LostItemFeeSection.test.js

```diff
@@ -0,0 +1,138 @@
+  const dash = '-';
+  const lostItemSectionAccordionLabel = 'ui-circulation.settings.lostItemFee.lostItemSection';
+  const displayYes = 'ui-circulation.settings.lostItemFee.yes';
+  const displayNo = 'ui-circulation.settings.lostItemFee.no';
+  const actualCost = 'ui-circulation.settings.lostItemFee.actualCost';
+  const lostItemReturnedCharged = 'ui-circulation.settings.lostItemFee.lostItemChargeOverdueBased';
+  const lostItemReturnedRemove = 'ui-circulation.settings.lostItemFee.lostItemRemoveOverdueFines';
+  const lostItemFeeSectionProps = {
+    formatMessage: jest.fn(() => null),
+    getPeriodValue: jest.fn((value) => value || dash),
+    lostItemFeeSectionOpen: true,
+    policy: {
+      chargeAmountItem: {
+        amount: 0,
+        chargeType: 'actualCost',
+      },
+      chargeAmountItemPatron: true,
+      chargeAmountItemSystem: true,
+      description: 'Test lost item fee policy',
+      feesFinesShallRefunded: {
+        duration: undefined,
+        intervalId: noop,
+      },
+      id: 'policyId',
+      itemAgedLostOverdue: {
+        duration: noop,
+        intervalId: noop,
+      },
+      lostItemChargeFeeFine: {
+        duration: 2,
+        intervalId: 'Days',
+      },
+      lostItemProcessingFee: 0,
+      lostItemReturned: 'Charge',
+      metadata: {
+        createdByUserId: 'userId',
+        createdDate: '2021-09-01',
+      },
+      name: 'Lost item fee policy',
+      patronBilledAfterAgedLost: {
+        duration: noop,
+        intervalId: noop,
+      },
+      patronBilledAfterRecalledItemAgedLost: {
+        duration: noop,
+        intervalId: noop,
+      },
+      recalledItemAgedLostOverdue: {
+        duration: noop,
+        intervalId: noop,
+      },
+      returnedLostItemProcessingFee: true,
+      replacedLostItemProcessingFee: true,
+      replacementProcessingFee: 0,
+      replacementAllowed: true,
+    },
+    viewLostItemFeeSection: true,
+  };
+  const lostItemFeeSectionAlternativeProps = {
+    ...lostItemFeeSectionProps,
+    policy: {
+      ...lostItemFeeSectionProps.policy,
+      chargeAmountItem: {
+        amount: 25,
+        chargeType: 'otherCost',
+      },
+      chargeAmountItemPatron: false,
+      chargeAmountItemSystem: false,
+      feesFinesShallRefunded: {
+        duration: 1,
+        intervalId: 1,
+      },
+      lostItemReturned: 'Remove',
+      returnedLostItemProcessingFee: false,
+      replacedLostItemProcessingFee: false,
+      replacementAllowed: false,
+    }
+  };
+
+  describe('with lostItemFeeSectionProps', () => {
+    beforeEach(() => {
+      render(<LostItemFeeSection {...lostItemFeeSectionProps} />);
+    });
+
+    it('should render LostItemFeeSection component', () => {
+      expect(screen.getByTestId('viewLostItemFeeSectionTestId')).toBeVisible();
+    });
+
+    it('should render open accordion with label', () => {
+      expect(screen.getByTestId('viewLostItemFeeSectionAccordionTestId')).toBeVisible();
```

Comment:
let us store `screen.getByTestId('viewLostItemFeeSectionAccordionTestId')` to variable to reuse its value